packages/property-grid/package.json

```diff
@@ -0,0 +1,97 @@
+    "build": " tsc 1>&2 && npm run copy:assets && npm run pseudolocalize",
+    "clean": "rimraf lib",
+    "lint": "tslint -p . 1>&2",
+    "test": "",
+    "cover": ""
+  },
+  "keywords": [
+    "Bentley",
+    "Property Grid React",
+    "iModel Property Grid",
+    "iModel.js"
+  ],
+  "author": {
+    "name": "Bentley Systems, Inc.",
+    "url": "http://www.bentley.com"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/imodeljs/viewer-components-react/tree/master/packages/property-grid"
+  },
+  "peerDependencies": {
+    "@bentley/bentleyjs-core": "^2.0.0",
+    "@bentley/context-registry-client": "^2.0.0",
+    "@bentley/frontend-authorization-client": "^2.0.0",
+    "@bentley/geometry-core": "^2.0.0",
+    "@bentley/imodelhub-client": "^2.0.0",
+    "@bentley/imodeljs-common": "^2.0.0",
+    "@bentley/imodeljs-frontend": "^2.0.0",
+    "@bentley/presentation-components": "^2.0.0",
+    "@bentley/imodeljs-i18n": "^2.0.0",
+    "@bentley/imodeljs-markup": "^2.0.0",
+    "@bentley/imodeljs-quantity": "^2.0.0",
+    "@bentley/itwin-client": "^2.0.0",
+    "@bentley/orbitgt-core": "^2.0.0",
+    "@bentley/presentation-common": "^2.0.0",
+    "@bentley/presentation-frontend": "^2.0.0",
+    "@bentley/product-settings-client": "^2.0.0",
+    "@bentley/ui-abstract": "^2.0.0",
+    "@bentley/ui-components": "^2.0.0",
+    "@bentley/ui-core": "^2.0.0",
+    "@bentley/ui-framework": "^2.0.0",
+    "@bentley/ui-ninezone": "^2.0.0",
+    "@bentley/webgl-compatibility": "^2.0.0"
+  },
+  "devDependencies": {
+    "@bentley/bentleyjs-core": "^2.0.0",
+    "@bentley/build-tools": "^2.0.0",
+    "@bentley/context-registry-client": "^2.0.0",
+    "@bentley/frontend-authorization-client": "^2.0.0",
+    "@bentley/presentation-components": "^2.0.0",
+    "@bentley/geometry-core": "^2.0.0",
+    "@bentley/imodelhub-client": "^2.0.0",
+    "@bentley/imodeljs-common": "^2.0.0",
+    "@bentley/imodeljs-frontend": "^2.0.0",
+    "@bentley/imodeljs-i18n": "^2.0.0",
+    "@bentley/imodeljs-markup": "^2.0.0",
+    "@bentley/imodeljs-quantity": "^2.0.0",
+    "@bentley/itwin-client": "^2.0.0",
+    "@bentley/orbitgt-core": "^2.0.0",
+    "@bentley/presentation-common": "^2.0.0",
+    "@bentley/presentation-frontend": "^2.0.0",
+    "@bentley/product-settings-client": "^2.0.0",
+    "@bentley/ui-abstract": "^2.0.0",
+    "@bentley/ui-components": "^2.0.0",
+    "@bentley/ui-core": "^2.0.0",
+    "@bentley/ui-framework": "^2.0.0",
+    "@bentley/ui-ninezone": "^2.0.0",
+    "@bentley/webgl-compatibility": "^2.0.0",
+    "@types/classnames": "^2.2.3",
+    "@types/i18next": "^8.4.2",
+    "@types/react": "~16.9.35",
+    "@types/react-dom": "^16.8.0",
+    "@types/react-redux": "^7.0.1",
+    "cpx": "^1.5.0",
+    "rimraf": "^3.0.2",
+    "tslint": "^5.11.0",
+    "typescript": "~3.7.4"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "i18next": "^10.2.2",
+    "react": "^16.8.0",
```

Comment:
The `react` and `react-dom` dependencies should probably be peers.  That's how most of the other shared packages are setup.